test/harvest/test_dimensions.py

```diff
@@ -97,6 +101,54 @@ def test_query_with_retry(mock_dimensions_dsl_query_error, caplog):
     )
 
 
+@pytest.fixture
+def mock_dimensions_dsl_query_login_error(monkeypatch):
```

Comment:
This is mostly the same as another pre-existing response fixture, but I wasn't sure how to extract just parts of this for the login scenario.